frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -440,6 +440,7 @@ class DashboardHeader extends Component {
             <DashboardHeaderButton
               icon="info"
               isActive={isShowingDashboardInfoSidebar}
+              data-testid="dashboard-header-info-button"
```

Comment:
this `data-testid` is unused and in general it is better to assign an `aria-label` to icon buttons

Reply:
575c7596f2215397f518010c9b05b84b32dd39b7